Source/UI/NewUI/PaymentForm/Factory/PaymentFormFactory.swift

```diff
@@ -13,6 +13,7 @@ public struct PaymentFormFactory {
     // Ensure a consistent identifier is used for the monitoring of a journey
     Self.sessionCorrelationID = UUID().uuidString
     let logger = FramesEventLogger(environment: configuration.environment, getCorrelationID: { Self.sessionCorrelationID })
+    logger.log(.paymentJourneyStart(environment: configuration.environment))
```

Comment:
@ehab-al-cko , @deepesh-vasthimal-cko see PR description for why this is not testable in current call site syntax. Let me know if you want to move from
```swift
PaymentFormFactory. buildViewController

// TO

PaymentFormFactory(). buildViewController
```

that apparent minor change makes all the difference :)